lib/router.js

```diff
@@ -70,48 +70,46 @@ module.exports = (function() {
   obj.websocket = {};
   obj.cache = {};
   obj.add = {
-    get:add.bind(obj, 'get'),
-    patch:add.bind(obj, 'patch'),
-    post:add.bind(obj, 'post'),
-    put:add.bind(obj, 'put'),
-    delete:add.bind(obj, 'delete'),
-    options:add.bind(obj, 'options'),
-    websocket:add.bind(obj, 'websocket'),
-    default:function(func) { this.default = func; }.bind(obj)
+    get: add.bind(obj, 'get'),
+    patch: add.bind(obj, 'patch'),
+    post: add.bind(obj, 'post'),
+    put: add.bind(obj, 'put'),
+    delete: add.bind(obj, 'delete'),
+    options: add.bind(obj, 'options'),
+    websocket: add.bind(obj, 'websocket'),
+    default: function (func) { this.default = func; }.bind(obj),
   };
-  obj.process = async function(method, path, req, res) {
-    if(this[method][path]) {
+  obj.process = async function (method, path, req, res) {
+    if (this[method][path]) {
       this[method][path](req, res);
-      return;
     } else {
       let found = false;
-      Object.keys(this[method]).forEach((async function(e) {
-        let m = path.match(new RegExp(e));
-        if(m && m.length > 0 && found === false) {
+      Object.keys(this[method]).forEach((async (e) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Async arrow function has a complexity of 9. Maximum allowed is 8.](https://app.codacy.com/manual/Akkeris/controller-api/pullRequest?prid=5244943)